### PR TITLE
Added basic project template

### PR DESCRIPTION
**Pull request by tajobe**
Let me know if there are any glaring omissions from the template (or any places where I forgot to change a reference to SimpleHealthbars2 which is where this is templatized from)

### CODE REVIEW

File: .github/workflows/release-workflow.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      - name: Release
+        uses: docker://antonyurchenko/git-release:v3
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
+          DRAFT_RELEASE: "true"
+          ALLOW_TAG_PREFIX: "true"
+        with:
+          args: |
+            build/libs/simplehealthbars2-${{ steps.format-version.outputs.replaced }}.jar
+            build/libs/simplehealthbars2-${{ steps.format-version.outputs.replaced }}-all.jar
```
Comment (Puremin0rez) [inline .github/workflows/release-workflow.yml]: SimpleHealthBars2 reference at this line and the line above it

File: build.gradle.kts [hunk 2]
```diff
@@ -0,0 +1,117 @@
...
+import java.time.format.DateTimeFormatter
+
+plugins {
+    kotlin("jvm") version "1.3.61"
+    id("com.github.johnrengelman.shadow") version "5.2.0"
+    id("pl.allegro.tech.build.axion-release") version "1.10.3"
+    id("org.jlleitschuh.gradle.ktlint") version "9.1.1"
+}
+
+val repoRef = "SimpleMC\\/SimpleHealthbars2"
```
Comment (Puremin0rez) [inline build.gradle.kts]: SimpleHealthBars2 reference

File: src/main/resources/plugin.yml [hunk 3]
```diff
@@ -0,0 +1,6 @@
+name: KotlinPluginTemplate
+main: org.simplemc.plugintemplate.KotlinPluginTemplate
+version: 0.1.0
+api-version: "1.15"
+website: https://github.com/SimpleMC/KotlinPluginTemplate
```
Comment (Puremin0rez) [inline src/main/resources/plugin.yml]: 404 / Not the repositories URL

Might be intentional if you plan on renaming it?

**Review by Puremin0rez — CHANGES_REQUESTED** [review]
Finally, some good heckin' easy plugin creation

File: build.gradle.kts (hunk 2 above)
Comment (tajobe) [reply build.gradle.kts]: I took the time to pull it out into a val so I could change it and just didn't...

File: src/main/resources/plugin.yml (hunk 3 above)
Comment (tajobe) [reply src/main/resources/plugin.yml]: naa, bad copy+paste from fixing the main above

**Review by Puremin0rez — APPROVED** [review]

**Pull request merged**